microservices/product-search-export/config/bundles.php

```diff
@@ -2,6 +2,5 @@
 
 return [
     Symfony\Bundle\FrameworkBundle\FrameworkBundle::class => ['all' => true],
-    Symfony\Bundle\WebServerBundle\WebServerBundle::class => ['dev' => true],
```

Comment:
It looks like you haven't properly uninstalled the webserver - you still have `/.web-server-pid` included in the `.gitignore`... Why don't you use Flex?